com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1288,6 +1309,16 @@ internal void ShutdownInternal()
             m_StopProcessingMessages = false;
 
             ClearClients();
+
+            if (wasClient)
+            {
+                OnClientStopped?.Invoke(wasServer);
+            }
+            if (wasServer)
+            {
+                OnServerStopped?.Invoke(wasClient);
+            }
+            
```

Comment:
Standards Check Failure #2:
Remove the whitespaces on this line.

Reply:
Should be fixed now